pkgs/applications/misc/qdirstat/default.nix

```diff
@@ -28,7 +28,7 @@ mkDerivation {
     for i in src/SysUtil.cpp src/FileSizeStatsWindow.cpp
     do
       substituteInPlace $i \
-        --replace /usr/bin/xdg-open ${xdg-utils}/bin/xdg-open
+        --replace /usr/bin/xdg-open xdg-open
```

Comment:
Is it possible to use makeWrapper here instead? This will break opening links for anyone who doesn't have `xdg-utils` explicitly installed.

Also, I made sure to verify the source code. It looks like it's being passed to `QProcess::startDetached` which thankfully supports reading from `$PATH` when not given a full path.

https://github.com/shundhammer/qdirstat/blob/1.8/src/FileSizeStatsWindow.cpp#L499